pop-api/src/v0/assets/fungibles.rs

```diff
@@ -323,9 +324,46 @@ pub fn burn(id: AssetId, account: AccountId, value: Balance) -> Result<()> {
 		.call(&(id, account, value))
 }
 
+/// The interface for managing metadata of fungible assets. It includes the PSP-22 Metadata interface
+/// for querying metadata.
+// TODO: if reviewers agree with replacing metadata related apis here, the dispatchable index has to
+//  be changed (quick fix).
 pub mod metadata {
 	use super::*;
 
+	/// Set the metadata for a token with a given asset ID.
+	///
+	/// # Parameters
+	/// - `id`: The identifier of the asset to update.
+	/// - `name`: The user friendly name of this asset. Limited in length by `StringLimit`.
+	/// - `symbol`: The exchange symbol for this asset. Limited in length by `StringLimit`.
+	/// - `decimals`: The number of decimals this asset uses to represent one unit.
+	///
+	/// # Returns
```

Comment:
Personally dont see the value in the returns section, especially based on the fact that the description is obvious and so is the return type. A concern would be that if it is started, it needs to be done for every public function for consistency, bring wasted efforts in terms of implementation and reviews.

Reply:
Agree, removed